renku/command/update.py

```diff
@@ -117,35 +121,44 @@ def _is_activity_valid(activity: Activity, plan_gateway: IPlanGateway, client_di
     return plan.invalidated_at is None
 
 
-def _get_modified_activities_and_paths(repository, activity_gateway) -> Tuple[Set[Activity], Set[str]]:
-    """Return latest activities that one of their inputs is modified.
+def get_all_modified_and_deleted_activities_and_entities(
+    repository, activity_gateway
+) -> Tuple[Set[Tuple[Activity, Entity]], Set[Tuple[Activity, Entity]]]:
+    """
+    Return latest activities that one of their inputs is modified/deleted along with the modified/deleted input entity.
```

Comment:
```suggestion
    Return latest activities with at least one modified or deleted input along with the modified/deleted input entity.
```
Current formulation sounds a bit off